File: server/projects/profile/src/main/java/com/raxim/myscoutee/profile/data/dto/rest/Like.java
package com.raxim.myscoutee.profile.data.dto.rest;

import com.fasterxml.jackson.annotation.JsonProperty;

public class Like {
    @JsonProperty(value = "from")
    private String from;

    @JsonProperty(value = "to")
    private String to;

    @JsonProperty(value = "rate")
    private Integer rate;

    @JsonProperty(value = "ref")
    private String ref;

    public Like(String from, String to, Integer rate, String ref) {
        this.from = from;
        this.to = to;
        this.rate = rate;
        this.ref = ref;
    }

    public String getFrom() {
        return from;
    }

    public void setFrom(String from) {
        this.from = from;
    }

    public String getTo() {
        return to;
    }

    public void setTo(String to) {
        this.to = to;
    }

    public Integer getRate() {
        return rate;
    }

    public void setRate(Integer rate) {
        this.rate = rate;
    }

    public String getRef() {
        return ref;
    }

    public void setRef(String ref) {
        this.ref = ref;
    }
}

File: server/projects/profile/src/main/java/com/raxim/myscoutee/profile/data/dto/rest/Like.kt
package com.raxim.myscoutee.profile.data.dto.rest

import com.fasterxml.jackson.annotation.JsonProperty

class Like(
    @field:JsonProperty(value = "from")
    val from: String? = null,
    @field:JsonProperty(value = "to")
    val to: String? = null,
    @field:JsonProperty(value = "rate")
    val rate: Int? = null,
    @field:JsonProperty(value = "ref")
    val ref: String? = null
    )